src/ElasticsearchPhpHandler.php

```diff
@@ -62,6 +59,8 @@ private function createPsr7Request(array $ringPhpRequest)
         $ringPhpRequest['headers'][$hostKey][0]
             = parse_url($ringPhpRequest['headers'][$hostKey][0])['host'];
 
+        $ringPhpRequest['headers']['Content-Type'] = 'application/json';
```

Comment:
Shouldn't this header have been set before the request was handed off to the HTTP handler? 

Also, I believe only requests with bodies should have a content-type header.

Reply:
Without *Content-Type: application/json* Elastic client throws next exception:
Elasticsearch\Common\Exceptions\BadRequest400Exception
Message: Content-Type header is missing

Reply:
That implies that the unsigned request would not have been accepted even if authentication were disabled, so I don't think it belongs in this library. 

Is there a later handler that is adding this header?